### ISSUE TITLE
Purpose of addFirstDatapoint

### ISSUE THREAD

**Issue by VishalNehra** [TeamAmaze/AmazeFileManager#1324]
@EmmanuelMess can you please tell the purpose of [AbstractProgressiveService#addFirstDatapoint](https://github.com/TeamAmaze/AmazeFileManager/blame/df434649bd87092c99b892d66540e9df0ce078e0/app/src/main/java/com/amaze/filemanager/asynchronous/services/ProgressiveService.java#L30) and the difference between ````addDatapoint````.
Tagging #1222

**Comment by EmmanuelMess** [comment]
I can remember this was part of some refractoring I did a while ago, I don't really know why everything was done the way it was, I just made the implementation a bit cleaner.

It seems that the original name for this method was `setInitDataPackage()`. The comment I forgot to copy was "Setting initial package to initialize charts in process viewer properly".

Maybe ask [yourself](https://github.com/TeamAmaze/AmazeFileManager/commit/c163cba9c0e6929deaae8d2ef367517349352005#diff-031d2d8d34248847af8d323f1e88aa60R577)?
